src/Character.js

```diff
@@ -109,8 +109,11 @@ class Character extends EventEmitter
    * @return {number}
    */
   getMaxAttribute(attr) {
-    const attribute = this.attributes.get(attr);
-    return this.effects.evaluateAttribute(attribute);
+    if (this.hasAttribute(attr)) {
+      const attribute = this.attributes.get(attr);
```

Comment:
You don't have to change this but just for future reference my preference here is "return early" style. Meaning you would invert the condition and `return null` inside the if. Then do the real logic outside of the if.